packages/earth-map/src/sagas/places/index.ts

```diff
@@ -37,10 +37,7 @@ import { LOCATION_QUERY } from '../model';
 import { getGroup, getPlaces } from 'sagas/saga-utils';
 
 export default function* places() {
-  yield all([
-    takeLatest(setPlacesSearch, searchPlaces),
-    takeLatest(nextPlacesPage, nextPage)
-  ]);
+  yield all([takeLatest(setPlacesSearch, searchPlaces), takeLatest(nextPlacesPage, nextPage)]);
```

Comment:
I sure hope this is the prettier doing this because it sure doesn't look prettier 😃